app/javascript/src/Stats/inlinechart/index.jsx

```diff
@@ -0,0 +1,93 @@
+// @flow
+
+import React, {Component} from 'react';
+import c3 from 'c3';
+import 'whatwg-fetch';
+import 'core-js/es6/promise';
+import moment from 'moment';
+import type {Props, State, RefObject, Data} from './types/index';
+
+class InlineChart extends Component<Props, State> {
```

Comment:
We should avoid extending from React Components (classes). Now it's even cleaner with the new https://reactjs.org/docs/hooks-overview.html. IMO, this component could be functional and using the new `effect hook` if you don't want to keep the data fetch in a different module.

Reply:
@didierofrivia `useState` is only available in **React v16.7.0-alpha**, do you think we should upgrade React now, or maybe it's a good idea to wait until a stable version is released? What do you think @josemigallas @3scale/system

Reply:
React Hooks are just a proposal at the moment.. IMHO it wouldn't be safe to start using them.

Reply:
What is the versioning policy they follow?
Is it a major or minor or patch upgrade?

In general, “we should wait for stable version” is a safe bet.

Reply:
meh